_data/risc_outgoing.yml

```diff
@@ -67,7 +67,17 @@
   description: >
     Login.gov pushes this event when a user changes the email address on their account.
   payload_schema:
-    *email_schema
+    - key: subject
+      description: An event should will a **subject** object, with the following keys
+      properties:
+        - key: subject_type
+          type: string
+          description: >
+            Will be **email**
+        - key: email
+          type: string
+          description: >
+            The new email address that was added, or the the email address that was removed.
```

Comment:
**comment:** I don't think this is right, based on the [spec](https://openid.net/specs/openid-risc-event-types-1_0-ID1.html#rfc.section.2.5) the `email` key should be the old value and there should be another key at the same level as `subject` called `new-value` that contains the new email that was added.

That said, since we allow users to include multiple emails I'm actually not sure how to handle that case (since no email is removed). Maybe `email` can be null? @jmhooper @mitchellhenke any thoughts?

Also, it looks like we're using `Identifier Recycled` incorrectly - the spec says (emphasis mine):

> Identifier Recycled signals that the identifier specified in the subject was recycled **and now it belongs to a new user**.

This doesn't seem relevant to us until someone adds an email that previously belonged to someone else (or I guess deletes and recreates their account). Should we even be sending it at all?

Beyond that, there's this whole section in the `Identifier Changed` specification that makes me think we should only be using `Recovery Information Changed` for email updates (and that event does not include the email itself).

> This event SHOULD be issued only by the provider that is authoritative over the identifier. For example, if the person that owns john.doe@example.com goes through a name change and wants the new john.row@example.com email then only the email provider example.com SHOULD issue an Identifier Changed event as shown in the example below.
>
> If an identifier used as a username or recovery option is changed, at a provider that is not authoritative over that identifier, then [Recovery Information Changed](https://openid.net/specs/openid-risc-event-types-1_0-ID1.html#recovery-information-changed) [recovery-information-changed] SHOULD be used instead.

Reply:
cc @NatashaJIbrahim @julialeague @mmagsa @nprimak for awareness

Reply:
Lun frla

Reply:
Hii